test/model_tests/user/user_info_test.dart

```diff
@@ -185,7 +185,7 @@ void main() {
 
     test('Check if Hive storage works', () async {
       Hive
-        ..init("./data")
+        ..init("./temporaryPath")
```

Comment:
previously it was storing in seprate folder named data and now it will store it in ./temporaryPath

Reply:
@palisadoes @noman2002

Reply:
This won't be a valid path for a Windows based developer. Any other suggestions?

Reply:
I checked on my Windows machine and it was working 

![image](https://user-images.githubusercontent.com/95746190/215884900-09b98c5f-0664-4b63-a42b-79d80dfb8261.png)

This method of storing hive file is used by other test also
for example
https://github.com/PalisadoesFoundation/talawa/blob/76f15534aa9887219da108bebad8afdc01364cbe/test/widget_tests/after_auth_screens/app_settings/app_setting_page_test.dart#L92-L99
and 
https://github.com/PalisadoesFoundation/talawa/blob/76f15534aa9887219da108bebad8afdc01364cbe/test/widget_tests/after_auth_screens/profile/edit_profile_page_test.dart#L79-L86

Reply:
OK, so we are using Hive to temporarily store data for testing, but in the `lib/` directory. Should we create a test specific const for this for consistency? Like in here https://github.com/PalisadoesFoundation/talawa/tree/develop/test/helpers

Reply:
Yes we can do something like this for consistency, maybe write all the hive init in one file and call them according to their requirement, instead of initialising Hive in each test case separately.
It will make a code more DRY also

Reply:
if you like this idea, I'll like to complete this. maybe in this PR or or in seperate issue

Reply:
Please fix it in this PR